app/models.py

```diff
@@ -102,14 +102,16 @@ def set_location_details(self, data, is_remote):
 
     def set_remote_details(self, data):
         self.remote_url = data["url"]
-        self.remote_joining_instructions = data["joining_instructions"]
+        if data["joining_instructions"]:
+            self.remote_joining_instructions = data["joining_instructions"]
```

Comment:
I can't work out how this works! Where do you set/default to None?

Reply:
It sets it as none automatically :)